packages/vanilla/src/sass/components/_modal.scss

```diff
@@ -1,146 +1,69 @@
-//@use '../base/colors' as *;
-
-// Future CSS to use for hidden components
-.cbp-hidden {
-  display: none;
-}
-
-/* this is added to the body when a dialog is open */
-.has-modal {
-  overflow: hidden;
-}
-
-.cbp-modal__backdrop {
-  background-color: rgba(0, 0, 0, 0.3);
-  display: none;
-  position: fixed;
-  overflow-y: auto;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-}
-
-.cbp-modal__backdrop.active {
-  display: block;
-}
-
-.cbp-modal, [role="dialog"] {
-  position: fixed;
-  left: 50%;
-  bottom: 2rem;
-  transform: translateX(-50%);
+.cbp-modal,
+dialog,
+[role='dialog'] {
   background-color: var(--cbp-color-white);
-  border-radius: 5px;
-  width: 336px;
-  height: fit-content;
-  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
-  
-  & > hr {
-    margin: 0px 16px;
-    border: 1px solid #A9AEB1; // TODO - token
-  }
-}
-
-.cbp-modal__header {
-  display: flex;
-  align-items: center;
-  padding-top: 20px;
-  padding-left: 16px;
-  padding-right: 16px;
-  padding-bottom: 16px;
-  color: var(--cbp-color-text-darker);
+  border-radius: var(--cbp-border-radius-softest);
+  border-width: 0;
+  box-shadow: var(--cbp-shadow-level-1-down);
+  width: 22rem;
```

Comment:
A fixed width of 22rem seems very heavy-handed and may like a bad idea for forms in a modal.

Reply:
leaving this fixed width for now.